### ISSUE TITLE
[Feature Request] Add containers groups, and goup oppening shortcuts or context menu button.

### ISSUE THREAD

**Issue by Esdeves** [mozilla/multi-account-containers#1048]
A must have would be having groups of containers, and the possibility to open a whole group in on go, with shortcuts (didn't find shortcuts to this point), or context menu button. 

Yeah. There are some people with this type of need, linking their stuffs as SQU
                                                                                                                ARE as possible.

Regards.

**Comment by SoftVision-PaulOiegas** [comment]
Hi @Esdeves and thanks for your report. If you can consider a container to be a group, for example, you create a new container named "Cars". Then all the tabs opened from that container can be hidden and shown as many times as you like trough a functionality already existent in the add-on.
See [this screen recording](https://goo.gl/oSFHK8) of the functionality that I'm describing.
Isn't this somehow what you need?

**Comment by Esdeves** [comment]
Hi !
Nope.

What I call a group would be having besides you "Cars" container, having one named "Planes", another "Bicycles", a third "Velociraptors", each four having for example a different mail account from the same provider, and being able to open the four different containers in the same time by clicking on a "Vehicles" group button in the context menu when we click on the mail provider bookmark.

I know in English, I'm sometime confused when I want to explain something a bit technical. Do I make sense right now ? ^^

**Comment by SoftVision-PaulOiegas** [comment]
Yes you do, but that would be a bit of inception, smaller containers in a bigger container. 😄 However, I see groovecoder (one of the Containers devs) has added some encouraging labels. Will see where this idea goes. 
Thanks for the details!

**Comment by Esdeves** [comment]
Ok. I didn't thought of this as bigger containers thought, just configurable shortcuts to open more than one container in one time. I may have misunderstood the feature.
